api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -301,6 +301,11 @@ type VeleroConfig struct {
 	// Disable informer cache for Get calls on restore. With this enabled, it will speed up restore in cases where there are backup resources which already exist in the cluster, but for very large clusters this will increase velero memory usage. Default is false.
 	// +optional
 	DisableInformerCache *bool `json:"disableInformerCache,omitempty"`
+	// Number of workers in worker pool for processing item backup. This will allow multiple items within
+	// a Velero backup to be backed up at the same time which may improve performance for backups with
+	// a large number of items. Default is 1.
+	// +optional
```

Comment:
Can we add `// +kubebuilder:validation:Minimum=1` to validate here?

Reply:
@mateusoliveira43 actually, on further thought, we don't want this since it will break upgrades, as it turns this into a required field. The current PR already checks value and ignores the field if it's <1.

Reply:
should we fail DPA reconcile if invalid value is passed then?

Reply:
I think if you made this an pointer to *int and then the `omitempty` would make it optional and only validate when it's present ?

Another option would be I think to:
```
// +kubebuilder:default=1
// +kubebuilder:validation:Minimum=1
```

This will make default backwards-compatible, but needs some testing if I am right here.

Reply:
not sure how "Minimum=1" interacts with pointer vals -- in particular, is that compatible with a pointer val of nil?
I think leaving the default as 0 without any requirements may be easier. I don't see any huge advantage of adding kubebuilder-level validation here  vs. just only passing values on to velero if they're >0.

Reply:
+1 on no real benefit, so without it will be fine

Reply:
@mateusoliveira43 Currently we're not failing reconcile, we're just ignoring it if they put something less than 1. If it's zero, that just means the user probably didn't specify anything, so ignoring is completely appropriate. I suppose we could check for negative values and fail reconcile if they are provided, but that's a bit of a silly edge case. If a user wants negative work done, then ignoring the request seems fine to me. But if people really want to check for negative values and error out, I can add that.